filebeat/module/elasticsearch/audit/_meta/fields.yml

```diff
@@ -22,10 +26,18 @@
       description: "The name of the action that was executed"
       example: "cluster:monitor/main"
       type: keyword
+    - name: url.params.text
+      description: "REST URI parameters"
+      example: "{username=jacknich2}"
+      type: text
```

Comment:
text? Was this intended to be a multifield?

Reply:
Yeah, lets make this a multi-field. Changing...